backends/gdx-backend-robovm-metalangle/res/META-INF/robovm/ios/robovm.xml

```diff
@@ -0,0 +1,10 @@
+<config>
+    <frameworks>
+        <framework>MetalANGLE</framework>
+        <framework>Metal</framework>
```

Comment:
We need a similar robovm.xml that has only the GLES framework in for GLES robovm, and we should openGL, MetalANGLE, and Metal out of the gdx-setup robovm.xml. (I think I saw it briefly on your other branch originally pushed, but seems to be missing here)

Reply:
As far as I know OpenGLES is already linked by default and wouldn't need to be specified explicitly. Therefor it seems a bit unnecessary to have a robovm.xml file that only defines a framework, that is not needed.

Reply:
Yeah, I mean we don't want GLES linked on the metal backend, and likewise for the other. So it makes sense to remove this from setup robovm.xml (which has gles already), and then make a robovm.xml for the gles robovm backend that declares it.

I think force link is fine to stay in the setup, as that doesn't change for each backend

Reply:
I thought since I'm already on it to reduce the setup robovm.xml, it maybe would be better to have also the other framework/forcelink declaration where they come from, namely the gdx native library.

Reply:
I think most belong on the backends, not the native lib, as the backend is the one that is dictating the java classes used, and the dependencies on the frameworks, but that requires duplicating them for both.